frontend/app/templates/work_packages/tabs/_attachments_table.html

```diff
@@ -1,4 +1,11 @@
-<table ng-if="attachments.length" class="detail-pane-table attachments-table">
+<table ng-if="attachments.length" class="attributes-table attachments-table">
+  <colgroup>
+    <col style="width: 20%">
+    <col style="width: 10%">
+    <col style="width: 20%">
+    <col style="width: 15%">
+    <col style="width: 25%">
```

Comment:
@NobodysNightmare When there is only one usage, I think inlining styles makes more sense.